bootstrap/main.py

```diff
@@ -0,0 +1,194 @@
+# Copyright 2018 Open Source Foundries Limited.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+'''West's bootstrap/wrapper script.
+'''
+
+import argparse
+import os
+from os.path import join
+import importlib
+import platform
+import subprocess
+import sys
+
+MANIFEST = 'manifest'
+MANIFEST_DEFAULT = 'https://github.com/zephyrproject-rtos/manifest'
+MANIFEST_REV_DEFAULT = 'master'
+WEST = 'west'
+WEST_DEFAULT = 'https://github.com/zephyrproject-rtos/west'
+WEST_REV_DEFAULT = 'master'
+if platform.system() == 'Windows':
+    # On Windows, the sentinel is hidden with attributes, rather than a
+    # leading '.', and thus doesn't contain one.
+    WEST_HIDDEN_SENTINEL = 'west-top'
```

Comment:
We need to be confident that this filename won't show up by accident in any of the repositories we manage, so it's worth picking its name carefully; any other ideas for what it should be named welcome.

Reply:
I would actually call it `.west` in all platforms and use the attributes on Windows to hide it while maintaining the same name

Reply:
Missed this comment at first, sorry. All comments now addressed.